src/ProjectTemplates/test/Templates.Tests/ApiTemplateTest.cs

```diff
@@ -35,7 +35,7 @@ public async Task ApiTemplateCSharp()
         await ApiTemplateCore(languageOverride: null);
     }
 
-    [ConditionalFact(Skip = "Unskip when there are no more build or publish warnings for native AOT.")]
+    [ConditionalFact(Skip = "Unskip when Helix supports native AOT. https://github.com/dotnet/aspnetcore/pull/47247/")]
```

Comment:
Nit: Should we update the tests in the gRPC templates with an issue reference as well?

https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/test/Templates.Tests/GrpcTemplateTest.cs

Reply:
The Grpc template tests are getting updated in that PR already. We can just wait for that PR to get clean.